src/redux/data.ts

```diff
@@ -1041,7 +1041,7 @@ export const addressAssetsReceived = (
   if (!isValidMeta) return;
   const { accountAddress, network } = getState().settings;
   const responseAddress = message?.meta?.address;
-  const addressMatch = accountAddress === responseAddress;
+  const addressMatch = toLower(accountAddress) === toLower(responseAddress);
```

Comment:
I think we recently prefer to avoid lodash where possible. Instead of an additional function call this si the same as:
```suggestion
  const addressMatch = accountAddress?.toLowerCase() === responseAddress?.toLowerCase();
```